lib/src/block_syntaxes/unordered_list_with_checkbox_syntax.dart

```diff
@@ -2,16 +2,9 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-import '../patterns.dart';
-import 'list_syntax.dart';
+import 'unordered_list_syntax.dart';
 
-/// Parses unordered lists.
-class UnorderedListWithCheckBoxSyntax extends ListSyntax {
-  @override
-  RegExp get pattern => ulWithCheckBoxPattern;
-
-  @override
-  String get listTag => 'ul_with_checkbox';
-
-  const UnorderedListWithCheckBoxSyntax();
+/// Parses unordered lists with checkboxes.
+class UnOrderedListWithCheckboxSyntax extends UnorderedListSyntax {
```

Comment:
One nit here: all just particular capitals 🤣 

Thanks for changing `CheckBox` to `Checkbox`, but also we should keep `Unordered` as is. It is one word.